app/models/claim.rb

```diff
@@ -194,6 +194,10 @@ def approvable?
     submitted? && !payroll_gender_missing? && !decision_made? && !payment_prevented_by_other_claims?
   end
 
+  def decision
+    decisions.order(created_at: :desc).first
```

Comment:
This is the same as:
```suggestion
    decisions.last
```